src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String string;
```

Comment:
The variable's name should not be the same as the class name of this variable because it's insufficient and doesn't represent its content. You've nicely renamed "variable" -> "year". Please do something similar here.